.travis.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       script: ./gradlew javadocs
 
     - stage: snapshot
-      if: branch = master AND NOT (type = pull_request)
+      if: (branch = master OR branch =~ /^r[0-9]*\.[0-9]*/) AND NOT (type = pull_request)
```

Comment:
It should be `+ `, not `*` as we want at least one digit.

Reply:
Fixed as suggestions